src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private static final Random RANDOM = new Random();

```

Comment:
```suggestion
    private final Random random = new Random();
```
let's not overuse static - you will be using java with Spring Boot, and we don't use static to inject dependencies there
also, beware that final + static doesn't always mean constants (it's debatable, some say object must be immutable too)